ci-scripts/unit-tests.sh

```diff
@@ -76,6 +76,14 @@ echo "Running schematics unit tests and code coverage for tracking library"
 
 yarn --cwd feature-libs/tracking run test:schematics --coverage
 
+echo "Running unit tests and code coverage for pickup-in-store library"
+
+ng test pickup-in-store --source-map --watch=false --code-coverage --browsers=ChromeHeadless
```

Comment:
use `--no-watch` and `--browsers ChromeHeadless`
check these:
https://angular.io/cli#boolean-options
https://angular.io/cli#array-options